src/components/popup/container.tsx

```diff
@@ -3,49 +3,44 @@ import { Flex, color } from '@stacks/ui';
 import { SettingsPopover } from './settings-popover';
 import { useRecoilValue } from 'recoil';
 import { hasRehydratedVaultStore } from '@store/wallet';
-import { Header } from '@components/header';
 
 interface PopupHomeProps {
-  title?: string;
-  onClose?: () => void;
-  hideActions?: boolean;
+  header?: any;
 }
 
-export const PopupContainer: React.FC<PopupHomeProps> = memo(
-  ({ children, title, onClose, hideActions }) => {
-    const hasRehydratedVault = useRecoilValue(hasRehydratedVaultStore);
+export const PopupContainer: React.FC<PopupHomeProps> = memo(({ children, header }) => {
+  const hasRehydratedVault = useRecoilValue(hasRehydratedVaultStore);
```

Comment:
Should this also have its own hook?

Reply:
Not sure, but probably will with the larger state refactor/improvements. outside of this PR scope